shared/src/common/types/user/user-delete-request-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type UserDeleteRequestRaramsDto = {
```

Comment:
```suggestion
type UsersDeleteRequestRaramsDto = {
```

Reply:
It's a typo? Shouldn't it be UsersDeleteRequest**Params**Dto?